src/cudadecoder/cuda-online-pipeline-dynamic-batcher.h

```diff
@@ -126,6 +127,7 @@ class CudaOnlinePipelineDynamicBatcher {
   std::vector<const std::string *> partial_hypotheses_;
   std::vector<bool> end_points_;
   std::atomic<std::uint32_t> n_chunks_not_done_;
+  std::map<CorrelationID, int> n_chunks_per_corr_;
```

Comment:
In C++ 11, `std::unordered_map` is preferred over `std::map` in most cases.

Reply:
Fixed